### ISSUE TITLE
Uncollapsed stack trace and infinite loading page under test environment

### ISSUE THREAD

**Issue by fightmaster** [symfony/symfony#23682]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Symfony version  | 3.3.5

Step to reproduce:
1. Try to open non-exist page under dev environment, you see collapsed stack trace. It's OK.
2. Try to open non-exist page under test environment (toolbar should be disabled, profiler: collect: false OR web_profiler: toolbar: false). You see uncollapsed stack trace and infinite loading page.

I installed the symfony-demo project and reproduce the issue too.

Some of the browsers display a pop-up window:

> A script on this page may be busy, or it may have stopped responding. You can stop the script now, open the script in the debugger, or let the script continue.

```
              // Prevents from disallowing clicks on links inside toggles /
                    var toggleLinks = document.querySelectorAll('.sf-toggle a');
                    for (var i = 0; i < toggleLinks.length; i++) {
                        addEventListener(toggleLinks[i], 'click', function(e) {
                            e.stopPropagation();
                        });
                    }
```

Regards,
Dmitry

**Comment by NicoHaase** [comment]
This was already fixed in #23577

**Comment by fightmaster** [comment]
@NicoHaase, oh, sorry. Yesterday, I search the same problem and did not find :(
